app/main.py

```diff
@@ -16,4 +16,16 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [
+        my_favourite_films,
```

Comment:
The issue is not resolved, your pull request should not contain files other than `app/main.py`. Visit a Q&A session to resolve this issue and request another review after other files are not present.